cmd/grit/clone.go

```diff
@@ -38,6 +40,23 @@ func clone(cfg grit.Config, idx *index.Index, c *cli.Context) error {
 	case transport.ErrEmptyRemoteRepository:
 		fmt.Fprintln(os.Stderr, "cloned an empty repository")
 
+		r, err := git.PlainInit(dir, false /* isBare */)
+		if err != nil {
+			return err
+		}
+
```

Comment:
There needs to be something that removes `dir` if creating the remote or the default branch fail.

Reply:
Added the `os.RemoveAll(dir)` in